test/help.t

```diff
@@ -83,6 +84,21 @@ class TestHelp(TestCase):
         code, out2, err2 = self.t("track -h")
         self.assertEqual(out1, out2)
 
+    def test_help_additional_info(self):
+        """Test that there are man pages for all the additional help pages."""
```

Comment:
I would try to keep the test naming scheme "timew ... should ...", e.g. "timew should display additional help pages"

Reply:
I should have noticed I wasn't following the pattern. I'll update and resolve this conversation when I update this pull request pending the question about what to actually test for below.